cockatrice/src/deckview.cpp

```diff
@@ -1,5 +1,6 @@
 #include <QApplication>
 #include <QGraphicsSceneMouseEvent>
+#include <QDebug>
```

Comment:
This is a WIP, so I have debugging information in here for my purposes. Will all be removed at the end.